src/pages/GE_TokenizeCard.page

```diff
@@ -63,6 +79,9 @@
                 // Reject any messages from an unexpected origin
                 return;
             } else if (event.data && event.data.action === 'createToken') {
+                // TODO - remove once elevate can process the gateway id
```

Comment:
@vmartinov-sforg I'm curious about the timing of this feature and how/when we remove it. I think part of the desire to have Elevate available in the package is to enable our broader team to demo to customers - so even though this isn't available to our broader customers we may want to be careful about this kind of visible artifact in support of testing. Not sure what the actual level of concern here is, will check with product

Reply:
@ptewson-sfdo Checking with product sounds good. We could remove all the visible items as soon as QE is completed.